src/core/lib/layer.js

```diff
@@ -537,7 +537,8 @@ export default class Layer {
     if (flags.updateTriggersChanged && !changeFlags.updateTriggersChanged) {
       changeFlags.updateTriggersChanged = flags.updateTriggersChanged;
       log.log(LOG_PRIORITY_UPDATE + 1,
-        () => `updateTriggersChanged: ${flags.updateTriggersChanged} in ${this.id}`);
+        () => `updateTriggersChanged: 
+        ${Object.keys(flags.updateTriggersChanged).join(', ')} in ${this.id}`);
```

Comment:
Logging: Doing it like that will generate a newline and lot of white space in this log message. You could use backslash and no indent on second line, or turn this into a braced function and calculate the keys in a temp variable.

setting deck.log.priority = 2 will let you test this.